packages/framework/fluid-framework/api-report/fluid-framework.legacy.public.api.md

```diff
@@ -461,8 +461,6 @@ TSchema
 // @public
 export type InsertableTypedNode<TSchema extends TreeNodeSchema, T = UnionToIntersection<TSchema>> = (T extends TreeNodeSchema<string, NodeKind, TreeNode | TreeLeafValue, never, true> ? NodeBuilderData<T> : never) | (T extends TreeNodeSchema ? Unhydrated<TreeNode extends NodeFromSchema<T> ? never : NodeFromSchema<T>> : never);
 
-export { InteriorSequencePlace }
```

Comment:
This and the two below are `@legacy + @alpha` (separate report) so correct to disappear?